airflow/www/security.py

```diff
@@ -307,28 +306,6 @@ def get_accessible_dags(self, user_action, user, session=None):
 
         return session.query(DagModel).filter(DagModel.dag_id.in_(resources))
 
-    def get_accessible_dag_ids(self, username=None):
```

Comment:
get_accessible_dag_ids() returns a SET, while get_readable_dag_ids() returns a LIST. This may bring some impacts.

Reply:
Thanks @XD-DENG, good catch. Updated.